wsd/COOLWSD.cpp

```diff
@@ -1533,6 +1558,10 @@ class RemoteConfigPoll : public RemoteJSONPoll
         }
         return booleanFlag.toString();
     }
+
+private:
+    // keeps track of remote config layer
+    AutoPtr<AppConfigMap> PersistConfig = nullptr;
```

Comment:
Better, but why the new naming convention? We use `_camelCase` for class members.

Also, it's good to prefix the namespace (`Poco::` in this case). We use namespaces everywhere for a reason (it makes maintenance easier, avoid clashes, helps easily identify the context, etc.).